plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -240,4 +264,29 @@ class JavaLocalLambdaRunConfigurationIntegrationTest(private val runtime: Lambda
         expectedOutput = input.uppercase(),
         addBreakpoint = { projectRule.addBreakpoint() }
     )
+
+    private fun createMinimalGradleBuildSetup(testProjectDir: File) {
+        val buildGradleFile = File(testProjectDir, "build.gradle.kts")
+        buildGradleFile.writeText(
+        """
+            plugins {
+                java
+            }
+ 
+            repositories {
+                mavenCentral()
+            }
+        """.trimIndent()
+        )
+    }
+
+    private fun createGradlePropertiesFile(gradleUserHome: File) {
+        val gradlePropertiesFile = File(gradleUserHome, "gradle.properties")
```

Comment:
shouldn't this be the project root? how would a gradle instance discover this location if the configuration is self-referential with nothing outside referencing it?